src/utils/search/project-search.js

```diff
@@ -32,3 +33,21 @@ export function findSourceMatches(source, queryText) {
   matches = [].concat(...matches);
   return matches;
 }
+export function highlightMatches(lineMatch) {
```

Comment:
so far, utils haven't had JSX, i think i'd prefer if this function were in `components/projectsearch/textsearch/utils/highlight.js`